demo/src/app/examples/other/observable-select/data.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { of } from 'rxjs/observable/of';
+import { Observable } from 'rxjs/Observable';
+
+@Injectable()
+export class DataService {
+    sports = [
+        { id: '1', name: 'Soccer' },
+        { id: '2', name: 'Basketball' },
+    ];
+
+    getSports(): Observable<any[]> {
+        return of(this.sports);
+    }
+
+}
```

Comment:
add a new line

Reply:
What do you think about adding Prettier? 😄.
I could set that up [in the PR I'm working for the lib setup](https://github.com/juristr/ng-formly/tree/nx-workspace-setup)

Reply:
why not!

Reply:
Ok, will include it in the setup then. PR is coming end of week/next week probably.

Reply:
take your time!